ksqldb-functional-tests/src/main/java/io/confluent/ksql/test/tools/ExpectedRecordComparator.java

```diff
@@ -140,7 +140,7 @@ private static boolean compareNumber(final Object actualValue, final JsonNode ex
     }
 
     if (actualValue instanceof Double) {
-      return expected.doubleValue() == (Double) actualValue;
+      return Math.abs(expected.doubleValue() - (Double) actualValue) < 0.000_001;
```

Comment:
six digits of comparison is pretty good for floating point math

Reply:
Could use http://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/number/IsCloseTo.html